DevelopGitHub/GitHubExtension/GitHubExtension.Activity.Internal.WebApi/Queries/GetActivityTypeQuery.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Linq;
+
+using GitHubExtension.Activity.DAL;
+
+namespace GitHubExtension.Activity.Internal.WebApi.Queries
+{
+    public class GetActivityTypeQuery : IGetActivityTypeQuery
+    {
+        private readonly ActivityContext _activityContext;
+
+        public GetActivityTypeQuery(ActivityContext activityContext)
+        {
+            _activityContext = activityContext;
+        }
+
+        public ActivityType GetUserActivityType(string name)
```

Comment:
public static ActivityType GetUserActivityType(this IGetActivityTypeQuery getActivityTypeQuery, string name)
        {
            ActivityType userActivityType = getActivityTypeQuery.ActivitiesTypes.FirstOrDefault<ActivityType>(n => n.Name == name);

            return userActivityType;
        }